Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -492,7 +491,7 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
       }
 
       //Check the object is on the platform
-      expect(object.getY()).to.be(-30); // -30 = -10 (platform y) + -20 (object height)
+      expect(object.getY()).to.be(-30.75); // not exactly -30 because not rounding coordinates (-10 (platform) -20 (object))
```

Comment:
@D8H We're considering removing the "roundCoordinates" option because it was only applying in a specific case (when on the floor). What do you think of doing this?

Reply:
Indeed, I think this property only purpose is to try to avoid sprite blurring and don't really do a good job.
It'll be really good to get rid of it because it's hard to cover every case for both values of this property.

About this assertion, I think the platformer character should be aligned with the platform. Because in this case in particular, it will probably be rounded to -31 by the render and overlap the platform from one pixel.

I wrote something that could fix this for the moving platform issue. Please, can you cherry pick this commit and see if it fixes the tests: https://github.com/4ian/GDevelop/pull/2574/commits/ba3bf44791677f3e633c5648775750000d92b1b3

Reply:
Actually there is other commit that do side effect on this too, I'll check them to see which ones to use.